.github/workflows/check.yml

```diff
@@ -8,54 +8,52 @@ concurrency:
   group: ${{ github.ref }}
   cancel-in-progress: true
 jobs:
-  test-pdflatex-linux-windows:
+  test-pdflatex:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-latest]
-        ruby: [ '3.1', '3.2', '3.3' ]
+        os:
+          - ubuntu-latest
+          # TODO: re-enable test on MacOS once we have a way to install TeXlive
+          #- macos-latest
+          - windows-latest
+        ruby:
+         - '3.1'
+         - '3.2'
+         - '3.3'
     name: "${{ matrix.os }} / ruby ${{ matrix.ruby }}"
-    runs-on: ubuntu-latest
+    runs-on: "${{ matrix.os }}"
     steps:
       - uses: actions/checkout@v4
+
       - uses: ruby/setup-ruby@v1
         with:
-          ruby-version: ${{ matrix.ruby }}
+          ruby-version: "${{ matrix.ruby }}"
           bundler-cache: true
+
       - name: Setup dependencies
+        env:
+          BUNDLE_CLEAN: "true"
+          BUNDLE_FROZEN: "false"
         run: |
           bundle update --bundler
           bundle install
+
       - name: Install TeX Live
+        if: matrix.os != 'windows-latest'
         uses: zauguin/install-texlive@v3
         with:
           packages: >
             latex-bin scheme-basic
-      - name: ltx2any
-        run: |
-          ruby ltx2any.rb test/test.tex
-  test-pdflatex-win:
-    strategy:
-      fail-fast: false
-      matrix:
-        ruby: [ '3.1', '3.2', '3.3' ]
-    name: "windows-latest / ruby ${{ matrix.ruby }}"
-    runs-on: windows-latest
-    steps:
-      - uses: actions/checkout@v4
-      - uses: ruby/setup-ruby@v1
-        with:
-          ruby-version: ${{ matrix.ruby }}
-          bundler-cache: true
-      - name: Setup dependencies
-        run: |
-          bundle update --bundler
-          bundle install
+
       - name: Install MiKTeX
+        if: matrix.os == 'windows-latest'
         run: |
           choco install miktex --no-progress
           echo "C:\Program Files\MiKTeX\miktex\bin\x64" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8
```

Comment:
@koppor Wow, this is _slow_:

![image](https://github.com/reitzig/ltx2any/assets/4246780/3e6a329f-f1ea-4296-aa12-a80fa06a0d12)

Do you have a way handy to use caching on this? Is there a dedicated action for choco you can recommend?

Reply:
Unfortunately, it's also brittle; I see failures sometimes. :/

Reply:
Caching for `choco install` is still an open issue: https://github.com/chocolatey/choco/issues/2134. - And I am not aware of any other actions.